integration-libs/cds/src/profiletag/services/profiletag-event.service.spec.ts

```diff
@@ -107,6 +108,34 @@ describe('ProfileTagEventTracker', () => {
     expect(nativeWindow.Y_TRACKING.eventLayer.push).not.toHaveBeenCalled();
   });
 
+  it(`Should not add config parameters to the q array if PT is already loaded`, () => {
+    const profileTagLoaded$ = profileTagEventTracker.addTracker();
+    let subscription = profileTagLoaded$.subscribe();
+    const baseSite = 'electronics-test';
+    getActiveBehavior.next(baseSite);
+    subscription.unsubscribe();
+
+    expect(nativeWindow.Y_TRACKING.q[0][0]).toEqual({
+      ...mockCDSConfig.cds.profileTag,
+      tenant: mockCDSConfig.cds.tenant,
+      siteId: baseSite,
+      spa: true,
+    });
+    expect(nativeWindow.Y_TRACKING.eventLayer.push).not.toHaveBeenCalled();
+
+    // reset the mock correctly so that the existing script is detected
+    spyOn(mockedWindowRef.document, 'querySelector').and.returnValue(
+      {} as Element
+    );
+    // retrigger profile-tag
+    subscription = profileTagLoaded$.subscribe();
+    getActiveBehavior.next(baseSite);
+    subscription.unsubscribe();
+    // Add script should be called only once during initial addition and not 2 times.
```

Comment:
You can remove this comment or re-word it to something like Verifies if the script gets injected again

because the Add script function does get called twice. The second time, it just gets caught and does not render the script tag

Reply:
Removed it :)